### ISSUE TITLE
ffmpeg fails with uneven frame dimensions

### ISSUE THREAD

**Issue by DeinAlptraum** [HansiMcKlaus/AudioSpectrumVisualizer#4]
If ffmpeg is called for images with an uneven height or width it exits with an error of the following sort:

```
[libx264 @ 000002937a698c80] height not divisible by 2 (1920x417)
Error initializing output stream 0:0 -- Error while opening encoder for output stream #0:0 - maybe incorrect parameters such as bit_rate, rate, width or height
ffmpeg exited with a failure.
```

If `-va` is set, then we should either pad or crop the frames to even height and width and give out a warning, or exit and inform the user that the video export requires even dimensions.

**Comment by irgendwr** [comment]
I would be in favor of your first suggestion: using a 1-pixel padding and printing a warning if height or width are uneven.

Padding could be done via ffmpeg by passing `-vf pad="ceil(iw/2)*2:ceil(ih/2)*2:0:oh-ih:color=<color>"`
(see [pad doc](https://ffmpeg.org/ffmpeg-filters.html#pad), `<color>` should be the background-color as hex, see [color-syntax](https://ffmpeg.org/ffmpeg-utils.html#color-syntax))

**Comment by HansiMcKlaus** [comment]
Fixed in f301387.
ffmpegs creates a padding with background color and prints message: "Warning: Image height and or width is uneven. Applying padding."
